.env.dist

```diff
@@ -5,6 +5,8 @@ DB_NAME=mautictest
 DB_USER=root
 DB_PASSWD=
 MAUTIC_DB_PREFIX=test_
+MAUTIC_TABLE_PREFIX=test_
```

Comment:
I think that this should be reverted. There was a series of PRs where we were fixing the DB prefixes with @mollux like https://github.com/mautic/mautic/pull/11496. This row was removed. Same for the local.php file change below.

Reply:
I removed the change on this file and made the github action remove the cache directly after composer install